src/components/calendar/days-view/days-view.ts

```diff
@@ -481,8 +482,14 @@ export default class IgcDaysViewComponent extends EventEmitterMixin<
     const last = weeks.length - 1;
 
     for (const [idx, week] of weeks.entries()) {
+      let hidden: boolean | undefined;
+      if (idx === 0 || idx === last) {
+        hidden =
+          week.every((day) => this.getDayProperties(day, today).hidden) ||
+          undefined;
+      }
       yield html`
-        <div role="row" part="days-row">
+        <div role="row" part="days-row" aria-hidden=${ifDefined(hidden)}>
```

Comment:
No point going the undefined route for the aria-hidden attribute.
Something along the lines of:
```ts
for (const [idx, week] of weeks.entries()) {
      let hidden = false;
      if (idx === 0 || idx === last) {
        hidden = week.every((day) => this.getDayProperties(day, today).hidden);
      }

      yield html`
        <div role="row" part="days-row" aria-hidden=${hidden}>
          ${this.showWeekNumbers
            ? this.renderWeekNumber(week[0], idx === last)
            : nothing}
          ${week.map((day) => this.renderDay(day, today))}
        </div>
      `;
    }
```